src/risclog/sqlalchemy/db.py

```diff
@@ -77,12 +77,16 @@ def using_bind(self, name):
         return session
 
 
-def get_database(testing=False):
+def get_database(testing=False, keep_session=False, expire_on_commit=False):
     """Get or create the database utility."""
+    if testing and not keep_session and not expire_on_commit:
+        # Keep the old testing behaviour by default
```

Comment:
This is seems generally like a good idea, but I would rather have that as the default parameter in the `Database.__init__()`. Or is it important, that we set these values here already?